lib/use_cases/create_response_use_case.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:survey/api/graphql/mutation/create_response_mutation_input.dart';
+import 'package:survey/repositories/survey_repository.dart';
+
+import 'base_use_case.dart';
+
+class CreateResponseUseCase extends UseCase<bool, ResponseInput> {
```

Comment:
perhaps `CreateSurveyResponseUseCase` is better to understand the context of response in this case 😄